dashboard/app/controllers/lti/v1/account_unlinking_controller.rb

```diff
@@ -0,0 +1,21 @@
+require 'metrics/events'
+
+module Lti
+  module V1
+    class AccountUnlinkingController < ApplicationController
+      before_action :authenticate_user!
+      # POST /lti/v1/account_linking/unlink
+      def unlink
```

Comment:
Is there a specific reason for creating a separate controller for this action?
If the reason is `#authenticate_user!`, we could add the condition to the callback:
```ruby
module Lti
  module V1
    class AccountLinkingController < ApplicationController
      before_action :authenticate_user!, only: %i[unlink]

      # POST /lti/v1/account_linking/unlink
      def unlink
        ...
      end
    end
  end
end
```

Reply:
Oh good call, that would also avoid having to hardcode the route to the separate controller. I'll move it to the linking controller.